src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int shortBase;
+    private int longBase;
+    private int height;
+
+    public IsoscelesTrapezoid(int shortBase, int longBase, int height) {
+        this.shortBase = shortBase;
+        this.longBase = longBase;
+        this.height = height;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new IsoscelesTrapezoid(getRandomInt(), getRandomInt(), getRandomInt());
+    }
```

Comment:
Remove the `createRandomFigure()` method from the `IsoscelesTrapezoid` class. This behavior should be in `FigureSupplier` class. Each figure class shouldn't be responsible for creating a random instance of itself.